students/Ben_freeman/lesson05/test.py

```diff
@@ -0,0 +1,30 @@
+# pylint: disable=unused-import, unused-wildcard-import,
+# pylint: disable=too-few-public-methods, too-many-arguments, wildcard-import
+# pylint: disable=logging-format-interpolation, pointless-string-statement
+import sys
+import unittest
+import database as db
+from unittest.mock import patch
+
+
+class TestFunctions(unittest.TestCase):
+
+    def setUP(self):
+        with patch("builtins.input", side_effect="yes"):
```

Comment:
Check this code and make sure that's really doing what you think. _side_effect_ is treated like an iterable, which means that in this case, since you're passing it a string, the first call to _input()_ will return "y", the next one "e" and then "s". So you're not really clearing the database. This line should look like this:

with patch("builtins.input", side_effect=("yes",)):